### PR TITLE
Include advertisement to DFTK school 2022

### PR DESCRIPTION
**Pull request by mfherbst**
Put some advertisement for next year's [DFTK school](https://school2022.dftk.org) into the docs.

### CODE REVIEW

**Comment by antoine-levitt** [comment]
Feels a bit overkill to have it in the docs, maybe just in the index.html?

**Comment by mfherbst** [comment]
Ok, I made it more subtle, but also included something in the github repo.

File: docs/src/school.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# DFTK School 2022
+For more details and registration info see the [school's website](https://school2022.dftk.org).
```
Comment (antoine-levitt) [inline docs/src/school.md:2]: so just remove it?

Comment (mfherbst) [reply docs/src/school.md:2]: Ah that will just redirect. No-one will ever see this.

Comment (mfherbst) [reply docs/src/school.md:2]: it's just because documenter does not allow you to put links into the sidebar.

Comment (antoine-levitt) [reply docs/src/school.md:2]: Oh strange, ok

**Pull request merged**